analysis_options.yaml

```diff
@@ -102,6 +102,7 @@ linter:
     - unnecessary_overrides
     - unnecessary_parenthesis
     - unnecessary_statements
+    - unsafe_html
```

Comment:
I'm not sure this adds value because the linter package doesn't use the html library.

Reply:
Apologies.